Description: Use cross-build compatible macro for finding pkg-config.
 .
 The package uses AC_PATH_PROG to discover the location of pkg-config(1). This
 macro fails to select the correct version to support cross-compilation.
 .
 This patch changes it to use AC_PATH_TOOL.
 .
 Refer to https://bugs.debian.org/884798 for details.
Origin: other
Last-Update: 2020-11-10

=== modified file 'configure.ac'
--- a/configure.ac	2020-11-04 15:41:30 +0000
+++ b/configure.ac	2020-11-10 16:36:54 +0000
@@ -30,7 +30,7 @@
 AC_SUBST(LIBTOOL_DEPS)
 
 dnl Check for pkg-config
-AC_PATH_PROG(PKGCONFIG,[pkg-config])
+AC_PATH_TOOL(PKGCONFIG,[pkg-config])
 
 dnl Support of internationalization (i18n)
 AM_GNU_GETTEXT([external])

